docker/singularity

```diff
@@ -1,53 +1,73 @@
 Bootstrap: docker
-
-From: nvidia/cuda:10.1-cudnn7-devel-ubuntu16.04
+From: nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04
 
 %post
-CURDIR=`pwd`
-
-apt-get update
-apt-get install -y tmux nano git wget
-
-apt-get install -y --no-install-recommends make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+  CURDIR=$(pwd)
+  TZ=Etc/UTC && \
+    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
+    echo $TZ > /etc/timezone
 
+  apt-get update -y
+  apt-get install -y tmux nano git wget
+  apt-get install -y --no-install-recommends \
+    build-essential \
+    gfortran \
+    libssl-dev \
+    zlib1g-dev \
+    libbz2-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    wget \
+    curl \
+    llvm \
+    libncurses5-dev \
+    xz-utils \
+    tk-dev \
+    libxml2-dev \
+    libxmlsec1-dev \
+    libffi-dev \
+    liblzma-dev \
+    liblapack-dev \
+    libopenblas-dev \
+    libhdf5-dev
 
-export PYENV_ROOT=/opt/pyenv
-export PATH="/opt/pyenv/bin:$PATH"
-curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
-pyenv install 3.7.2
-echo 'export PATH=/opt/pyenv/versions/3.7.2/bin/:$PATH' >> $SINGULARITY_ENVIRONMENT
-export PATH=/opt/pyenv/versions/3.7.2/bin/:$PATH
+  export PYENV_ROOT=/opt/pyenv
```

Comment:
Any reason we have added spaces infront of all of these?

Reply:
Spaces were added for readability/styling. Styled the recipe similar to the Sylabs [example recipe](https://sylabs.io/guides/3.8/user-guide/definition_files.html).

If its preferred to keep the recipe the original way, say the word and I can revert it back.

Reply:
No preference from me, I was just curious :-)